### ISSUE TITLE
result on conll2003 fine tuning (no checkpoint)

### ISSUE THREAD

**Issue by manucarbonell** [studio-ousia/luke#78]
Hi, I am trying to reproduce the experiment results reported on the paper but cannot do so by fine tuning the Luke - large model. I do get the reported score using the checkpoint file but if I do the 5 epoch fine tuning the test F1 is slightly lower than the reported. Is there a specific seed to set to get the exact same results?
Thank you very much

**Comment by ikuyamada** [comment]
We trained the NER model with hyper-parameters specified in the code. Unfortunately, various factors can affect the performance such as differences between computational environments.

**Comment by manucarbonell** [comment]
I see. Thanks for the answer
